tests/publiccloud/run_ltp.pm

```diff
@@ -156,34 +149,34 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    # LTP command line preparation
-    # The python3-paramiko is too old (2.4 on 15-SP6)
-    # The python311-paramiko is from SLE-Module-Python3-15-SP5-Updates which we have in PC tools image
-    zypper_call("in python311-paramiko python311-scp");
+    assert_script_run("cd kirk");
+    assert_script_run("python3.11 -m venv env311");
+    assert_script_run("source env311/bin/activate");
+    assert_script_run("pip3.11 install asyncssh msgpack");
```

Comment:
We have the `python3-asyncssh` & `python3-msgpack` packages to avoid installing from pip.

I'd avoid installing from pip for security reasons specially if this VM contains cloud credentials.

Reply:
hmm, this can be workable, I'll try test it but with `python3.11-asyncssh` as `python3-*` packages on sle are python-3.6

Reply:
ah, not possible ... we don't have this packages in distro ..